k8s/examples/wordpress/README.md

```diff
@@ -0,0 +1,133 @@
+# Wordpress with MySQL on Kubernetes with StorageOS Persisent Storage
+
+This example shows how to deploy a WordPress site; with a MySQL database, an NGINX Ingress Controller and an NFS Server. The files build two StatefulSets that can be used after a StorageOS Cluster has been created. For
+more information on how to install a StorageOS cluster please see
+[the StorageOS documentation](https://docs.storageos.com/docs/introduction/quickstart)
+for more information.
+
+# Deploy
+
+## Prerequisites
+
+ 1. Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/) for your Provider.
+
+ > Check the Ingress Controller service is deployed and the pods are running
+ > using the following commands.
+
+ ```
+$ kubectl get service
+NAME                                       TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)                      AGE
+kubernetes                                 ClusterIP      10.43.240.1     <none>          443/TCP                      5m
+my-nginx-nginx-ingress-controller          LoadBalancer   10.43.244.81    35.195.97.236   80:30758/TCP,443:31772/TCP   5m
+my-nginx-nginx-ingress-default-backend     ClusterIP      10.43.240.77    <none>          80/TCP                       5m
+
+```
+> Make a note of the `EXTERNAL-IP`, as this will be used to access the WordPress Site.
+
+```
+$ kubectl get pods
+
+NAME                                                      READY   STATUS              RESTARTS   AGE
+my-nginx-nginx-ingress-controller-7f96c4cb75-xmp8h        1/1     Running             0          3m
+my-nginx-nginx-ingress-default-backend-5547cf6b98-cpkfb   1/1     Running             0          3m
+
+```
+
+2. Self-Signed Certificate for TLS
+
+ > When creating the certificate you will be asked to provide the
+ > Common Name (eg, your name or your server's hostname), in case of our example
+ > it needs to be `test.example.com` . The host is specified in `25-ingress.yml`.
+ ```
+$ mkdir certs
+$ cd certs
+$ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./tls-key.key -out ./tls-cert.crt
+$ kubectl create secret tls tls-certificate --key ./tls-key.key --cert ./tls-cert.crt
+$ kubectl label secret tls-certificate app=wordpress
+ ```
+
+3. Clone this repository.
+
+```
+$ git clone https://github.com/storageos/deploy.git storageos
+```
+
+## Deploy MySQL
+
+Firstly you will need to create all the Kubernetes objects for the MySQL part of this example.
+
+```
+$ cd storageos
+$ kubectl create -f ./k8s/examples/wordpress/mysql
+```
+
+Once this is done you can check that a mysql pod is running.
+```
+$ kubectl get pods -l app=wordpress,tier=mysql
+
+NAME                READY   STATUS    RESTARTS   AGE
+client              1/1     Running   0          4m
+wordpress-mysql-0   1/1     Running   0          4m
+
+```
+
+Connect to the MySQL Client Pod and connect to the MySQL server through the service.
+
+```
+$ k exec -it client -- bash 
+root@client:/# mysql -h wordpress-mysql -uroot -p
```

Comment:
You can just do the same without the client. The client is intended for demonstration purposes when we don't put an application in front of it. 

```
kubectl exec wordpress-mysql-0 -- /bin/bash
$ mysql -uroot -p
mysql> show databases;
```